docs/content/preview/yugabyte-voyager/reference/data-migration/export-data.md

```diff
@@ -99,8 +131,74 @@ The valid *arguments* for export data status are described in the following tabl
 | -e, --export-dir <path> | Path to the export directory. This directory is a workspace used to store exported schema DDL files, export data files, migration state, and a log file.|
 | -h, --help | Command line help. |
 
-### Example
+#### Example
 
 ```sh
 yb-voyager export data status --export-dir /dir/export-dir
 ```
+
+### get data-migration-report
+
+**Note** that this command is applicable for Live migrations only.
+
+Provides a consolidated report of data migration per table among all the databases involved in the live migration. The report includes the number of rows exported, the number of rows imported, change events exported and imported (INSERTS, UPDATES, and DELETES), and the final row count on the database.
+
+#### Syntax
+
+```text
+Usage: yb-voyager get data-migration-report [ <arguments> ... ]
+```
+
+#### Arguments
+
+The valid *arguments* for get data-migration-report are described in the following table:
+
+| Argument | Description/valid options |
+| :------- | :------------------------ |
+| -e, --export-dir <path> | Path to the export directory. This directory is a workspace used to store exported schema DDL files, export data files, migration state, and a log file.|
+| -h, --help | Command line help. |
+| --source-db-password <password>| Password to connect to the source database server. If you don't provide a password via the CLI during any migration phase, yb-voyager will prompt you at runtime for a password. Alternatively, you can also specify the password by setting the environment variable `SOURCE_DB_PASSWORD`. If the password contains special characters that are interpreted by the shell (for example, # and $), enclose it in single quotes. |
+| --source-replica-db-password <password> | Password to connect to the source-replica database server. Alternatively, you can also specify the password by setting the environment variable `SOURCE_REPLICA_DB_PASSWORD`. If the password contains special characters that are interpreted by the shell (for example, # and $), enclose it in single quotes. |
+| --target-db-password <password>| Password to connect to the target YugabyteDB server. Alternatively, you can also specify the password by setting the environment variable `TARGET_DB_PASSWORD`. If you don't provide a password via the CLI during any migration phase, yb-voyager will prompt you at runtime for a password. If the password contains special characters that are interpreted by the shell (for example, # and $), enclose the password in single quotes. |
```

Comment:
target YugabyteDB server - wait, the database or the server?

Reply:
I am not seeing the usage of server in the cli help .. Will change it to database.